src/System.Text.Encoding/tests/EncodingTestHelpers.cs

```diff
@@ -10,6 +11,12 @@ public static class EncodingHelpers
     {
         private static readonly RandomDataGenerator s_randomGenerator = new RandomDataGenerator();
 
+        public static IEnumerable<object[]> Encodings_TestData()
```

Comment:
The plan is only to have the negative (Invalid) tests in EncodingHelpers.cs
Positive tests will use the helper methods to ensure full coverage of all methods, but not actually have test data for positive tests